client/src/components/Header/DictionaryToggleButton.tsx

```diff
@@ -0,0 +1,24 @@
+import * as Icon from '../../assets/icons';
+import { useState } from 'react';
+
+function DictionaryToggleButton() {
+  const [dictionaryOn, setDictionary] = useState<boolean>(false);
```

Comment:
```suggestion
  const [dictionaryOn, setDictionaryOn] = useState(false);
```

dictionaryOn 상태는 true/false 타입만 사용할 것으로 추측되는데 default 값이 false이므로 ts에서는 **boolean으로 추론이 가능**해보여요! 그럼 <boolean> 이라고 명시하지 않아도 괜찮을것 같아요! 만약 중요한 값이고 명시하고 싶으면 상관 없을거 같아요

또한 useState 훅을 사용할떄 대부분 상태와 상태변경 함수 이름을 동일하게 가져가는 편이 많더라고요... `setDicitionaryOn`으로 해도 좋을것 같은데 이부분은 논의해봅시다!